axon-framework/modules.md

```diff
@@ -59,7 +59,7 @@ This module allows Axon Framework components to be configured in the Spring Appl
 
 ### Axon Spring Boot Starter
 
-This module provides Spring Boot auto-configuration for your project. It is by far the easiest option to get started as it automatically configures all Axon components. It is explained in more details [here](spring-boot.md).
+This module provides Spring Boot auto-configuration for your project. It is by far the easiest option to get started as it automatically configures all Axon components. It is explained in more details [here]().
```

Comment:
Similar question as earlier, but about the `spring-boot.md` link this time.

Reply:
Resolved